src/toolkits/object_detection/object_detector.cpp

```diff
@@ -205,6 +203,20 @@ void object_detector::init_options(
       FLEX_UNDEFINED,
       1,
       std::numeric_limits<int>::max());
+  options.create_integer_option(
+      /* name          */ "grid_height",
```

Comment:
n.b. I added the comments here in part to prevent clang-format from condensing these into one or two lines. The comments do make things a bit more readable, I think